coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5620,6 +5620,35 @@ def roll(context, node):
     context.add(x, node.name)
 
 
+def _construct_unfold_indices(N, C, H, W, kernel_size, stride):
+    """
+    The implementation below assumes x to be contiguous
```

Comment:
What's `x`?

Reply:
Ah that was copied from `im2col`. Revised to make it clearer when stand alone:
```
    A utility function to construct indices for torch.unfold (im2col),
    assuming the torch.unfold input `x` to be contiguous
```